integration-tests/testutils/common/common.go

```diff
@@ -39,8 +39,9 @@ import (
 const (
 	// BaseAltPartitionPath is the path to the B system partition.
 	BaseAltPartitionPath = "/writable/cache/system"
-	needsRebootFile      = "/tmp/needs-reboot"
-	channelCfgFile       = "/etc/system-image/channel.ini"
+	// NeedsRebootFile is the file that a test writes in order to request a reboot.
+	NeedsRebootFile = "/tmp/needs-reboot"
```

Comment:
@mvo5 The pattern is correct, but @elopio is right: won't work in this case because tests are looking at the package from the outside.